bin/hera/src/hera.rs

```diff
@@ -25,8 +26,19 @@ impl HeraCli {
     pub fn run(self) -> Result<()> {
         match self.subcmd {
             HeraSubCmd::ExEx(cli) => cli.run(|builder, args| async move {
-                let Some(cfg) = ROLLUP_CONFIGS.get(&args.l2_chain_id).cloned().map(Arc::new) else {
-                    bail!("Rollup configuration not found for L2 chain ID: {}", args.l2_chain_id);
+                let cfg = match &args.l2_config_file {
+                    Some(path) => {
+                        // try to load the rollup configuration from a file
+                        let file = File::open(path).wrap_err("Failed to open l2 config file")?;
+                        Arc::new(from_reader(file).wrap_err("Failed to read l2 config file")?)
+                    }
+                    None => {
+                        // try to load the rollup configuration from the registry by chain ID
+                        let Some(cfg) = RollupConfig::from_l2_chain_id(args.l2_chain_id) else {
+                            bail!("Failed to find l2 config for chain ID {}", args.l2_chain_id);
+                        };
+                        Arc::new(cfg)
```

Comment:
So this isn't going to work for many chains. The [`from_l2_chain_id`](https://github.com/ethereum-optimism/superchain-registry/blob/main/bindings/rust-primitives/src/rollup_config.rs#L320-L329) method only loads via the hardcoded rollup configs. We need to use the `superchain-registry's `ROLLUP_CONFIG` map which is dynamically generated at compile time.